resources/js/views/settings/roles/View.vue

```diff
@@ -0,0 +1,294 @@
+<template>
+  <div>
+    <h3>
+      {{ id === 'new' ? $t('settings.roles.new') : (
+        viewOnly ? $t('settings.roles.view', { name: $te(`app.roles.${model.name}`) ? $t(`app.roles.${model.name}`) : model.name })
+          : $t('settings.roles.edit', { name: $te(`app.roles.${model.name}`) ? $t(`app.roles.${model.name}`) : model.name })
+      ) }}
+    </h3>
+    <hr>
+
+    <b-form @submit='saveRole'>
+      <b-container fluid>
+        <b-form-group
+          label-cols-sm='3'
+          :label="$t('settings.roles.name')"
+          label-for='name'
+          :state='fieldState("name")'
+        >
+          <b-form-input id='name' type='text' v-model='model.name' :state='fieldState("name")' :disabled='isBusy || viewOnly'></b-form-input>
+          <template slot='invalid-feedback'><div v-html="fieldError('name')"></div></template>
+        </b-form-group>
+        <b-form-group
+          label-cols-sm='3'
+          :label="$t('settings.roles.roomLimit.label')"
+          label-for='room-limit'
+          :state='fieldState("room_limit")'
+        >
+          <b-form-input id='room-limit' type='number' :state='fieldState("room_limit")' v-model='model.room_limit' min='-1' :disabled='isBusy || viewOnly || model.room_limit === null || model.room_limit === -1'></b-form-input>
```

Comment:
![image](https://user-images.githubusercontent.com/4281791/94654359-3b4fd080-02fd-11eb-9a52-2cc5bf8ec8b9.png)

Creating a new role it is possible to select both checkboxes.
I would recommend a radio with 'System default', 'Unlimited', 'Custom amount'
Only if custom amount is selected the input field is shown below the radio.

Reply:
Done

Reply:
Please change the order of the radio and input field.

Reply:
Done